scripts/clean-cargo-tests.sh

```diff
@@ -3,12 +3,12 @@ ROOT_DIR=$(realpath "$SCRIPT_DIR/..")
 
 sudo nvme disconnect-all
 
-for c in $(docker ps -a --filter "label=io.mayastor.test.name" --format '{{.ID}}') ; do
+for c in $(docker ps -a --filter "label=io.composer.test.name" --format '{{.ID}}') ; do
```

Comment:
Just curious to understand, where the `.name` is used while creating the composer ?
TEST_LABEL_PREFIX is having the value `io.composer.test` which is label_prefix.

Reply:
We add that when creating the containers